azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java

```diff
@@ -59,6 +59,9 @@
     private static final String DEPLOY_START = "Trying to deploy artifact to %s...";
     private static final String DEPLOY_FINISH = "Successfully deployed the artifact to https://%s";
     private static final String SKIP_DEPLOYMENT_FOR_DOCKER_APP_SERVICE = "Skip deployment for docker app service";
+    private static final String NO_RUNTIME_CONFIG = "No runtime related configuration is specified in pom.xml. " +
```

Comment:
No runtime related configuration is in pom.xml=>You need to specified runtime in pom.xml for creating azure webapps